### ISSUE TITLE
Implement ability to load OSTIA sea ice data in pre-processor

### ISSUE THREAD

**Issue by simonrp84** [ORAC-CC/orac#5]
The data we currently use for sea ice extent is very low resolution (something like 1 degree) and poorly-suited to the pixel size of most imaging instruments. The Operational Sea Surface Temperature and Sea Ice Analysis is at 0.05 degree resolution, so implementing this will reduce the 'blockiness' of retrievals in ice-prone regions.

**Comment by caropoulsen** [comment]
Sounds interesting What data set is This ECMWF data??


________________________________

**Comment by simonrp84** [comment]
No. ECMWF data does contain a sea ice flag (that we can already read if we choose the right driver option) but that is also lower resolution (0.125 degree at best).
Some details about OSTIA are here: http://ghrsst-pp.metoffice.com/pages/latest_analysis/ostia.html

The Nowcasting SAF has implemented OSTIA within their cloud processing and - based on an afternoon of playing around with it - the addition of this product makes a nice difference there. Should be the same for us.

**Comment by caropoulsen** [comment]
excellent sounds good have you made the changes to the code to include it?

________________________________

**Comment by simonrp84** [comment]
No, I have not made the changes - this doesn't affect any of my work so it's low priority.
But I thought I'd raise the issue to remind me to look into it if I get some free time. The additions are non-trivial so don't expect it any time soon!

**Comment by simonrp84** [comment]
As part of this I have now included better snow albedo values, previously these were only defined for 4 wavelengths (0.67, 0.8, 1.6 and 3.9 micron). Now we have 40 wavelengths across VIS and IR. The most suitable albedo is found by interpolating the channel central wavelength to the closest wavelengths in the albedo database. Albedo values are from the ASTER spectral library.

Note: Ice is not included, only snow. Am searching for better ice values.

OSTIA work is still ongoing (slowly).

**Comment by simonrp84** [comment]
No longer developing new ORAC features, unassigning myself.
